pom.xml

```diff
@@ -118,6 +118,7 @@
         <module>features/servlet-mgt</module>
         <module>features/system-config-mgt</module>
         <module>features/registration-framework</module>
+        <module>components/async-status-mgt</module>
```

Comment:
shall we move the `<module>components/async-status-mgt</module>` to the end of components/ modules